android/app/src/main/res/layout/item_twitter_feed.xml

```diff
@@ -3,6 +3,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@color/feed_bg"
+    android:paddingBottom="@dimen/feed_item_profile_info_padd"
```

Comment:
Use standard spacing padding

Reply:
Actually this padding is equal to 10dp and the standard one is much greater than the 10dp